CMakeLists.txt

```diff
@@ -49,6 +53,16 @@ option(ENABLE_SANITIZE "Sanitize tool" OFF)
 option(ENABLE_SECURITY "Security Ford protocol protection" ON)
 option(ENABLE_HMI_PTU_DECRYPTION "Policy table update parsed by hmi" ON)
 option(ENABLE_EXTENDED_POLICY "Turns extended flow which requires embedded system interaction" ON)
+option(USE_COTIRE "Use Cotire to speed up build (currently only for commands tests)" ON)
```

Comment:
Both @JackLivio and I have seen this issue consistently when building on our Ubuntu Virtual Machines. I have 4GB of ram allocated to my VM.

Error Message:
```
Scanning dependencies of target commands_test_unity
[ 95%] Building CXX object src/components/application_manager/test/commands/CMakeFiles/commands_test_unity.dir/cotire/commands_test_CXX_unity.cxx.o
[ 95%] Linking CXX executable commands_test
/usr/lib/gcc/x86_64-linux-gnu/5/../../../x86_64-linux-gnu/crt1.o:function _start: error: undefined reference to 'main'
collect2: error: ld returned 1 exit status
src/components/application_manager/test/commands/CMakeFiles/commands_test_unity.dir/build.make:123: recipe for target 'src/components/application_manager/test/commands/commands_test' failed
make[2]: *** [src/components/application_manager/test/commands/commands_test] Error 1
CMakeFiles/Makefile2:4025: recipe for target 'src/components/application_manager/test/commands/CMakeFiles/commands_test_unity.dir/all' failed
make[1]: *** [src/components/application_manager/test/commands/CMakeFiles/commands_test_unity.dir/all] Error 2
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2
```